cmake/libtuv.cmake

```diff
@@ -48,6 +45,6 @@ set_property(DIRECTORY APPEND PROPERTY
 set(TUV_INCLUDE_DIR ${DEPS_TUV_SRC}/include)
 set(TUV_LIBS tuv)
 
-if("${TARGET_OS}" STREQUAL "LINUX")
+if(${TARGET_OS} STREQUAL "LINUX")
   list(APPEND TUV_LIBS pthread)
```

Comment:
Does android need this, too?

Reply:
There is no need to link pthread and rt separately on android.

Ref: https://developer.android.com/ndk/guides/stable_apis#clib